ssh/tailssh/incubator.go

```diff
@@ -435,13 +435,16 @@ func trySU(dlogf logger.Logf, ia incubatorArgs) (handled bool, err error) {
 		defer sessionCloser()
 	}
 
-	loginArgs := []string{"-l", ia.localUser}
+	loginArgs := []string{su,
+		"-w", "SSH_AUTH_SOCK", // pass through SSH_AUTH_SOCK environment variable to support ssh agent forwarding
+		"-l",
+		ia.localUser}
```

Comment:
style nit: I'd put `su,` and `ia.localUser,` on their own lines without touching the braces

Reply:
Ah yes, that's actually my preference too, not sure what I didn't do that here.